.github/workflows/build-and-deploy.yml

```diff
@@ -187,7 +187,7 @@ jobs:
           environment: review
           pr: ${{ github.event.pull_request.number }}
           sha: ${{ needs.build.outputs.image_tag }}
-          slack-webhook: ${{ secrets.SLACK_WEBHOOK }}
+          slack-webhook: ${{ steps.get-secret.outputs.slack_webhook_twd_findpub_tech }}
```

Comment:
This comes from outputs of the get-secret step. So the get-secret step must declare this secret as a key

Reply:
Reverted this now.